gms/versioned_value.hh

```diff
@@ -189,6 +194,10 @@ public:
             return versioned_value(make_full_token_string(tokens));
         }
 
+        versioned_value streams_timestamp(db_clock::time_point streams_ts) {
```

Comment:
Let's split the changes to gms/versioned_value.hh to a different commit.

Reply:
Done